CLI/support/sparkle_job_help.py

```diff
@@ -125,17 +128,75 @@ def get_running_jobs() -> list[SlurmRun]:
+    # \033 is the escape character (ESC) in ASCII
+    # [{lines}A is the escape sequence that moves the cursor up.
+    print(f"\033[{lines}A", end="")
+    # [J is the exape sequence that clears the console from the cursor down
+    print("\033[J", end="")
+
+
 def wait_for_all_jobs() -> None:
     """Wait for all active jobs to finish executing."""
-    running_jobs = get_running_jobs()
-    prev_jobs = len(running_jobs) + 1
-    while len(running_jobs) > 0:
-        if len(running_jobs) < prev_jobs:
-            print(f"Waiting for {len(running_jobs)} jobs...", flush=True)
-        time.sleep(10.0)
-        prev_jobs = len(running_jobs)
-        running_jobs = [run for run in running_jobs
-                        if run.status == Status.WAITING or run.status == Status.RUNNING]
+    jobs = get_running_jobs()
+    verbosity_setting = gv.settings.get_output_verbosity()
+    running_jobs = [run for run in jobs
+                    if run.status == Status.WAITING or run.status == Status.RUNNING]
+
+    # If verbosity is quiet there is no need for further information
+    if verbosity_setting == VerbosityLevel.QUIET:
+        prev_jobs = len(running_jobs) + 1
+        while len(running_jobs) > 0:
+            if len(running_jobs) < prev_jobs:
+                print(f"Waiting for {len(running_jobs)} jobs...", flush=True)
+            time.sleep(3.0)
+            prev_jobs = len(running_jobs)
+            running_jobs = [run for run in running_jobs
+                            if run.status == Status.WAITING
+                            or run.status == Status.RUNNING]
+
+    # If verbosity is standard the command will print a table with relevant information
+    elif verbosity_setting == VerbosityLevel.STANDARD:
+        # Collect dependencies and partitions for each job
+        jobs = get_dependencies(jobs)
+        while len(running_jobs) > 0:
+            # Information to be printed to the table
+            information = [["RunId", "Name", "Status", "Dependencies", "Finished Jobs"]]
+            running_jobs = [run for run in running_jobs
+                            if run.status == Status.WAITING
+                            or run.status == Status.RUNNING]
+            for job in jobs:
+                finished_jobs_count = sum(1 for status in job.all_status
+                                          if status == Status.COMPLETED)
+                information.append(
+                    [job.run_id,
+                     job.name,
+                     job.status,
+                     "None" if len(job.dependencies) == 0
+                        else ", ".join(job.dependencies),
+                     f"{finished_jobs_count}/{len(job.all_status)}"])
+            # Print the table
+            table = tabulate(information, headers="firstrow", tablefmt="grid")
+            print(table)
+            time.sleep(3)
+
+            # Clears the table for the new table to be printed
+            lines = table.count("\n") + 1
```

Comment:
Does the table/tabulate allow us to detect the number of lines in a clear way? For example, could the length of the information object yield the same information?

Reply:
I think this is the cleanest way
Theoretically we could also get the information from the information object, however only by counting how many extra lines are used for formatting and adding those lines (e.g. lines = len(information) * 2 + 3 or something like that)